src/Sylius/Component/Core/Resolver/DefaultShippingMethodResolver.php

```diff
@@ -33,27 +35,20 @@ class DefaultShippingMethodResolver implements DefaultShippingMethodResolverInte
     private $shippingMethodRepository;
 
     /**
-     * @var ZoneMatcherInterface|null
+     * @var ZoneMatcherInterface
      */
     private $zoneMatcher;
 
     /**
      * @param ShippingMethodRepositoryInterface $shippingMethodRepository
-     * @param ZoneMatcherInterface|null $zoneMatcher
+     * @param ZoneMatcherInterface $zoneMatcher
```

Comment:
Shouldn't we leave this class as it is (even if buggy)?

Reply:
nope, see [this comment](https://github.com/Sylius/Sylius/pull/9398#discussion_r193361059) 😄